modules/kargoBidAdapter.js

```diff
@@ -72,6 +65,10 @@ export const spec = {
       eids: firstBidRequest.userIdAsEids
     });
 
+    if(currency != null && currency != "USD") {
```

Comment:
We'd probably want to do something similar to the other objects/vars that we want to leave out of the object if they're null or default values.